Training/TPU-v5p/Llama2-7B-Maxtext/README.md

```diff
@@ -1 +1,3 @@
 # Instructions for training Llama2-7B-Maxtext on TPU v5p
+
+Please check the v4 user guide first: https://github.com/gclouduniverse/reproducibility/blob/main/Training/TPU-v4/Llama2-7B-Maxtext/TPU_v4_user_guide.md
```

Comment:
Nit: 

Please check the v4 user guide first: [here](https://github.com/gclouduniverse/reproducibility/blob/main/Training/TPU-v4/Llama2-7B-Maxtext/TPU_v4_user_guide.md)

Reply:
Done.